docs/_static/routes.csv

```diff
@@ -1,56 +1,55 @@
-Methods,Path,Backend parsers,Backend serializers,Frontend request,Frontend response
```

Comment:
Why did you delete some of the columns? Also `upload-contract` is still in the table.

Reply:
Hmm I ran `python -m app dev routes --csv-format` to generate it as per documented, maybe I missed something?

Reply:
Ah, those docs and/or the command needs to be updated to preserve the content. I suggest copying the file from `main` and just deleting the rows that are out-of-date.

Reply:
I fixed the command on main. You can merge main, copy over routes.csv, and run:

```
python -m app dev routes --csv-format --file docs/_static/routes.csv
```

Reply:
> and just deleting the rows that are out-of-date.

And then this?

Reply:
No longer needed, I don't think.